### ISSUE TITLE
tinkey: OutOptions.validate not called by CreatePublicKeysetCommand

### ISSUE THREAD

**Issue by Matir** [google/tink#546]
**Describe the bug**

When `--out` is not used with `tinkey create-public-keyset` a NullPointerException is raised.

**To Reproduce**

Generate a private key (of an asymmetric format) then try to create the public keyset without specifying `--out`.

```
/bazel-bin/tinkey/tinkey create-public-keyset --in ../../foo.json                                                                      ✘ 1  
Exception in thread "main" java.lang.NullPointerException                                                                                                                                     
        at com.google.crypto.tink.JsonKeysetWriter.write(JsonKeysetWriter.java:86)                                                                                                            
        at com.google.crypto.tink.CleartextKeysetHandle.write(CleartextKeysetHandle.java:79)                                                                                                  
        at com.google.crypto.tink.tinkey.CreatePublicKeysetCommand.create(CreatePublicKeysetCommand.java:47)                                                                                  
        at com.google.crypto.tink.tinkey.CreatePublicKeysetCommand.run(CreatePublicKeysetCommand.java:31)
        at com.google.crypto.tink.tinkey.Tinkey.main(Tinkey.java:55)
```

**Expected behavior**

Like other commands, it should print to stdout.

**Error messages, stack traces, etc.**

```
/bazel-bin/tinkey/tinkey create-public-keyset --in ../../foo.json                                                                      ✘ 1  
Exception in thread "main" java.lang.NullPointerException                                                                                                                                     
        at com.google.crypto.tink.JsonKeysetWriter.write(JsonKeysetWriter.java:86)                                                                                                            
        at com.google.crypto.tink.CleartextKeysetHandle.write(CleartextKeysetHandle.java:79)                                                                                                  
        at com.google.crypto.tink.tinkey.CreatePublicKeysetCommand.create(CreatePublicKeysetCommand.java:47)                                                                                  
        at com.google.crypto.tink.tinkey.CreatePublicKeysetCommand.run(CreatePublicKeysetCommand.java:31)
        at com.google.crypto.tink.tinkey.Tinkey.main(Tinkey.java:55)
```

**Version information**

 - Language: Java
 - Version: 1.6.1
 - Environment: Linux

**Additional context**

PR is on the way with a fix.

**Comment by thaidn** [comment]
Thanks! We'll fix this in 1.7.0.

**Comment by morambro** [comment]
This is now included in Tink 1.7.0
